src/sentry/static/sentry/app/components/assistant/guideAnchor.jsx

```diff
@@ -59,19 +57,68 @@ const GuideAnchor = createReactClass({
     let {target, type} = this.props;
 
     return (
-      <div
+      <span
         ref={el => (this.anchorElement = el)}
+        style={{position: 'relative'}}
         className={classNames('guide-anchor', type)}
       >
         {this.props.children}
-        <span
-          className={classNames(target, 'guide-anchor-ping', {
-            active: this.state.active,
-          })}
+        <StyledGuideAnchor
+          type={type}
+          className={classNames('guide-anchor-ping', target)}
+          active={this.state.active}
         />
-      </div>
+      </span>
     );
   },
 });
 
+const StyledGuideAnchor = styled('div')`
+  width: 12px;
+  height: 12px;
+  cursor: pointer;
+  z-index: 999;
+  position: relative;
+  pointer-events: none;
+  animation: ${expandOut} 1.5s ease-out infinite;
+  visibility: hidden;
+
+  &,
+  &:before,
+  &:after {
+    position: absolute;
+    display: block;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    background-color: ${p => p.theme.greenLight};
+    border-radius: 50%;
+  }
+
+  &:before,
+  &:after {
+    content: '';
+  }
+
+  &:before {
+    width: 75%;
+    height: 75%;
+    background-color: ${p => p.theme.greenDark};
+  }
+
+  &:after {
+    width: 50%;
+    height: 50%;
+    color: ${p => p.theme.green};
+  }
+
+  ${p => (p.active ? 'visibility: visible;' : '')} ${p =>
+      p.type == 'text'
```

Comment:
What are the other types?

Reply:
gonna remove this too. might wait to remove the actual attributes because there are a lot of them already, but i can do that in a follow-up pull.

Reply:
don't remove the target, it's used to link a step in the guide with the appropriate guide anchor. e.g. https://github.com/getsentry/sentry/blob/master/src/sentry/assistant/guides.py#L17

Reply:
uh, ^ was meant to be for your other comment about removing the target attr

Reply:
oh ok. so we just don't need it in the classname then? bc that's what i did anyways haha